### PR TITLE
waybar: add page

### PR DESCRIPTION
**Pull request by TechnoFlask**
Waybar is a highly customizable Wayland bar for Sway and Wlroots based compositors.

- [x] The page(s) are in the correct platform directories: `common`
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/linux/waybar.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# waybar
+
+> Highly customizable Wayland bar for Sway and Wlroots based compositors.
+> More information: <https://github.com/Alexays/Waybar>.
+
+- Start waybar with default configuration and stylesheet:
```
Comment (acuteenvy) [inline pages/linux/waybar.md:6]:
```suggestion
- Start `waybar` with the default configuration and stylesheet:
```

File: pages/linux/waybar.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Use a different configuration file:
+
+`waybar {{-c|--config}} {{path/to/config}}`
+
+- Use a different stylesheet file:
+
+`waybar {{-s|--style}} {{path/to/stylesheet}}`
+
+- Set the logging level for waybar:
```
Comment (acuteenvy) [inline pages/linux/waybar.md:18]:
```suggestion
- Set the logging level:
```

**Review by acuteenvy — COMMENTED** [review]
Waybar also works on BSDs - the page should be in `common`.

**Comment by TechnoFlask** [comment]
Thank you for your help. I will close this pr and add this file to common directory.

**Comment by spageektti** [comment]
You don't need to open a new PR. You can just move the file.

**Pull request closed without merging**